src/domain/certificates/useCases/displayHtmlHasNoWidthConstraint.ts

```diff
@@ -0,0 +1,15 @@
+const SECTION_TYPES: string[] = ['text', 'image', 'signature', 'variable'];
+
+function getMatchingElementsExpr (): string {
+  return SECTION_TYPES.map((sectionType: string) => `section[class*="${sectionType}"]`).join(', ');
+}
+
+export default function displayHtmlHasNoWidthConstraint (displayHtml: string): boolean {
```

Comment:
I wonder if that should also be an option from the API, which is by default false but that we activate where needed?

Reply:
Hmm the problem is that different BV instances might display legacy HTML, and we have to make sure the width constraint is applied even if the host is not enabling that API option.

Reply:
ok then